static/styles/zulip.scss

```diff
@@ -1447,7 +1447,7 @@ a:hover code {
 
     /* Ensure bulleted lists are nicely centered in 1-line messages */
     ul {
-        margin: 5px 0 5px 20px;
+        margin: -2px 0 -2px 20px;
     }
     &.rtl ul {
         margin: 5px 20px 5px 0;
```

Comment:
We need to update this as well.

Probably actually we should do a new first commit that changes this `rtl` (right-to-left languages like Arabic) formatting CSS to just set margin-left and margin-right, without listing values for top/bottom, so that we don't need to change that code for this PR.

Reply:
Added a commit for this prior to the commit with the discussed changes.